pyproject.toml

```
@@ -35,7 +35,7 @@ Ansible = "ansible_pygments.styles:AnsibleStyle"
 ansible = "ansible_pygments.styles:AnsibleStyle"
 
 [tool.poetry.dependencies]
-python = "^3.6.0"
+python = "^3.9.0"
```

Upper boundaries are discouraged here
```suggestion
python = ">= 3.9.0"
```